tools/spelling/spelling_dictionary.txt

```diff
@@ -1194,6 +1194,7 @@ unary
 unconfigurable
 undef
 underflowing
+underflows
```

Comment:
I don't see uses of this word, I think you can undo this addition to the dictionary.